src/components/Tokens/TokenTable/SearchBar.tsx

```diff
@@ -61,7 +61,7 @@ const SearchInput = styled.input<{ isExplore: boolean; isClicked?: boolean }>`
   }
 
   @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
-    width: ${({ isExplore, isClicked }) => (isExplore ? (isClicked ? 'min(100%, 200px)' : '0') : '100%')};
+    width: ${({ isExplore, isOpen }) => (isExplore ? (isOpen ? 'min(100%, 200px)' : '0') : '100%')};
```

Comment:
The NavTabs have gained some margin and now when the searchbar opens at medium breakpoints, `Transactions` wraps to a new line
![Screenshot 2023-10-04 at 10 37 25 AM](https://github.com/Uniswap/interface/assets/11512321/2138f0a3-ba1c-4956-af5d-d0ba95028f83)
![Screenshot 2023-10-04 at 10 40 52 AM](https://github.com/Uniswap/interface/assets/11512321/a07a10f7-e359-49db-9bce-347006e94b63)